### ISSUE TITLE
[CI] ObjectMapperTests.testMerge fails

### ISSUE THREAD

**Issue by csoulios** [elastic/elasticsearch#64607]
https://gradle-enterprise.elastic.co/s/4ovfay2fdedky/tests/:server:test/org.elasticsearch.index.mapper.ObjectMapperTests/testMerge#1

REPRODUCE WITH: 

```
./gradlew ':server:test' --tests "org.elasticsearch.index.mapper.ObjectMapperTests.testMerge" -Dtests.seed=A8F03A66D34E2BA8 -Dtests.security.manager=true -Dtests.locale=es-US -Dtests.timezone=MIT -Druntime.java=11
```

Error thrown:

```
Suite: Test class org.elasticsearch.index.mapper.ObjectMapperTests
  1> [2020-11-05T08:30:13,547][INFO ][o.e.i.m.ObjectMapperTests] [testMerge] before test
  1> [2020-11-05T08:30:14,516][INFO ][o.e.i.m.ObjectMapperTests] [testMerge] after test
  2> REPRODUCE WITH: ./gradlew ':server:test' --tests "org.elasticsearch.index.mapper.ObjectMapperTests.testMerge" -Dtests.seed=A8F03A66D34E2BA8 -Dtests.security.manager=true -Dtests.locale=es-US -Dtests.timezone=MIT -Druntime.java=11
  2> java.lang.AssertionError: expected:<STRICT> but was:<null>
        at __randomizedtesting.SeedInfo.seed([A8F03A66D34E2BA8:1B51E482DC959D3D]:0)
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at org.elasticsearch.index.mapper.ObjectMapperTests.testMerge(ObjectMapperTests.java:126)
  2> NOTE: leaving temporary files on disk at: /Users/csoulios/devel/elastic/elasticsearch/server/build/testrun/test/temp/org.elasticsearch.index.mapper.ObjectMapperTests_A8F03A66D34E2BA8-001
  2> NOTE: test params are: codec=Asserting(Lucene87): {}, docValues:{}, maxPointsInLeafNode=1933, maxMBSortInHeap=7.701096554778511, sim=Asserting(RandomSimilarity(queryNorm=false): {}), locale=es-US, timezone=MIT
  2> NOTE: Mac OS X 10.15.7 x86_64/Oracle Corporation 11.0.3 (64-bit)/cpus=16,threads=1,free=447336960,total=536870912
  2> NOTE: All tests run in this JVM: [ObjectMapperTests]
```

**Comment by csoulios** [comment]
Test failure may be related to #64543

**Comment by romseygeek** [comment]
Fixed by #64626
